cluster/swarm/cluster.go

```diff
@@ -216,7 +216,12 @@ func (c *Cluster) RemoveContainer(container *cluster.Container, force, volumes b
 // RemoveNetwork removes a network from the cluster
 func (c *Cluster) RemoveNetwork(network *cluster.Network) error {
 	err := network.Engine.RemoveNetwork(network)
-	c.refreshNetworks()
+	if err == nil {
+
+	}
+	for _, engine := range c.engines {
+		engine.DeleteNetwork(network)
```

Comment:
if we do that it's like before, it's going to hit every engine via the network and kill our perfs